switchlink/switchlink_db.c

```diff
@@ -80,7 +80,7 @@ switchlink_db_status_t switchlink_db_interface_add(
     uint32_t ifindex, switchlink_db_interface_info_t *intf_info) {
   switchlink_db_intf_obj_t *obj =
       switchlink_malloc(sizeof(switchlink_db_intf_obj_t), 1);
-  ovs_assert(obj);
+  krnlmon_assert(obj);
```

Comment:
You might want to make asserts of this sort `krnlmon_assert(obj != NULL)`. If the assertion fails, the expression within the parentheses is displayed as part of the message:
```
program: program.c:5: main: Assertion 'a != 1' failed.
```
`Assertion 'obj != NULL' failed` is more meaningful than `Assertion 'obj' failed`.